src/risknet/config/conf.yaml

```diff
@@ -0,0 +1,15 @@
+general:
```

Comment:
why is this coming in as a new thing in the git diff? didn't we already have this?

Reply:
Oh I realized that I had named it something else on github so I renamed the file